app/src/main/res/layout/item_stream_segment.xml

```diff
@@ -48,6 +48,18 @@
             app:layout_constraintTop_toTopOf="parent"
             tools:text="Lorem ipusum is widely used to create long sample text which is used here too" />
 
+        <TextView
+            android:id="@+id/textViewChannel"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:ellipsize="end"
+            android:maxLines="1"
+            android:textAppearance="@style/PlayQueueItemSubtitle"
+            android:textSize="@dimen/video_item_search_title_text_size"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
```

Comment:
I noticed the other view have this too. However the parent view is not a `ConstraintLayout` but a `LinearLayout`.

Reply:
Good catch. I did not notice that while copy-pasting the text view.